packages/bruno-app/src/components/Environments/EnvironmentSettings/DeleteEnvironment/index.js

```diff
@@ -27,7 +27,7 @@ const DeleteEnvironment = ({ onClose, environment, collection }) => {
           handleConfirm={onConfirm}
           handleCancel={onClose}
         >
-          Are you sure you want to delete <span className="font-semibold">{environment.name}</span> ?
+          <div className='flex gap-1'>Are you sure you want to delete <span className="font-semibold max-w-[200px] truncate inline-block">{environment.name}</span> ?</div>
```

Comment:
I think, we don't need to do this. We should display the full name in the delete modal